docs/review-checklist.md

```diff
@@ -1,3 +1,63 @@
+
+# Code style
+
+Code should conform to the project's coding style. We use Seastar's
+coding style, see seastar/coding-style.md.
```

Comment:
We actually have a coding-style.md in the Scylla project as well. It used to be nothing more than a reference to seastar/coding-style.md, but it now has one more thing.